content/02.body.md

```diff
@@ -0,0 +1,31 @@
+## Introduction
+
+### Node degree bias
```

Comment:
Should we call this section just "node degree" or have a separate section of "node degree". Again trying to get away from the impression that node degree only matters when it is biased in its deviation from the true degree.